### DISCUSSION TITLE
Use of this toolbox in ArcGIS Pro

### DISCUSSION THREAD

**Discussion by gsalomaomartins** [evanthoms/Cross-Section#15] [Q&A] [answered]
Hi, thanks for creating this toolbox. Does this work in ArcGIS Pro? I've tried to add the toolbox there but nothing happened... Appreciate any advice. Kind regards, Gabriela.

**Comment by ethoms-usgs** [comment, marked as answer]
No, sorry, it will not work in Pro. It's a different version of Python and toolbox format.

Take a look at the Project to Cross Section tool in the GeMS toolbox at this branch:

https://github.com/usgs/gems-tools-pro/tree/xs-update

You do not need to have your data in the GeMS schema to use the tool. It doesn't do everything that the toolbox at this repo does but it will get you started with a simple cross section panel frame.

**Reply by gsalomaomartins** [reply]
Thanks a lot. Will try the GeMS. Best wishes.
